docs_user/adoption-attributes.adoc

```diff
@@ -8,7 +8,10 @@ ifeval::["{build}" == "upstream"]
 :OpenStackShort: OSP
 :OpenShiftShort: OCP
 :osp_curr_ver: Antelope
+:osp_prev_ver: Wallaby
 :OpenStackInstaller: TripleO
+:CephService_first_ref: Manila
+:CephService: Manila
```

Comment:
`Manila` is not technically a `CephService`, it's an `OSP` component that has a `Ceph` backend in this case (NFS)

Reply:
Can we change the variable to "CephComponent"?

Reply:
ack, makes sense to me to distinguish between the two names using a variable. Can we use something like: `rhos_component_storage_file`?
- `:rhos_comp_storage_file: Manila` if upstream
- `:rhos_comp_storage_file: Shared File System service` if downstream